builtin/providers/aws/resource_aws_vpc.go

```diff
@@ -55,6 +55,13 @@ func resourceAwsVpc() *schema.Resource {
 				Computed: true,
 			},
 
+			"assign_amazon_ipv6_cidr_block": {
```

Comment:
Totally agreed :) Made the change to `assign_generated_ipv6_cidr_block` - i went with generated so it was closer to the AWS docs